resources/testdata/Cargo.toml

```diff
@@ -97,9 +97,12 @@ cldr_json_glob = [
     "cldr-core/supplemental/ordinals.json",
     "cldr-core/supplemental/plurals.json",
     "cldr-dates-full/main/$LOCALES/ca-gregorian.json",
+    "cldr-dates-full/main/$LOCALES/timeZoneNames.json",
     # Extra data for feature coverage in provider_cldr tests:
     "cldr-dates-full/main/cs/ca-gregorian.json",
+    "cldr-dates-full/main/cs/timeZoneNames.json",
     "cldr-dates-full/main/haw/ca-gregorian.json",
+    "cldr-dates-full/main/haw/timeZoneNames.json",
 ]
```

Comment:
Hmm, it appears that I need this. I get a compiler error saying that the `timeZoneNames.json` files for `cs` and `haw` are missing if I don't have them here.

Reply:
Okay, I think I see what's going on.  TimeZonesProvider walks the `cldr-dates-full` directory to find locales.  Since `haw` and `cs` are being pulled in for ca-gregorian.json, then those folders exist, and then TimeZonesProvider naively assumes that timeZoneNames.json should also be present in that directory.  See `impl TryFrom<&dyn CldrPaths> for TimeZonesProvider<'_>`.

Meh, I guess leave it for now.  I don't see a super quick fix that doesn't have side-effects; we want to bubble up `open_reader(&path)` errors, for example, because it might be an actual I/O error that we want to report.